template.typ

```diff
@@ -58,14 +57,13 @@
     align(
       center,
       [
-
         #image(logo, width: 6.5cm)
         #v(2em, weak: true)
 
-        #upper[Dipartimento di #text(department)]
+        #upper[#i18n.at(locale).department_of #text(department)]
```

Comment:
It would be better to make this completely customizable, as someone could want to use "Computer Science Department" instead of "Department of Computer Science"

Reply:
yup, totally agree. I'll make it an argument of the project.